modules/passkey/test/GasBenchmarkingProxy.spec.ts

```diff
@@ -75,15 +79,22 @@ describe('Gas Benchmarking [@bench]', function () {
         })
 
         const { x, y } = decodePublicKey(credential.response)
-        const verifier = await verifiers[key].getAddress()
+        const verifier = verifiers[key].target
 
-        await factory.createSigner(x, y, verifier)
-        const signer = await ethers.getContractAt('SafeWebAuthnSigner', await factory.getSigner(x, y, verifier))
+        const isValidSignatureInterface = new ethers.Interface(['function isValidSignature(bytes32,bytes) external view returns (bytes4)'])
+
+        const precompileAddress = `0x${'00'.repeat(18)}0100`
+        await setCode(precompileAddress, await ethers.provider.getCode(verifier))
+
+        const verifiersData = BigInt(ethers.solidityPacked(['uint32', 'address'], [Number(precompileAddress), verifier]))
+
+        await proxyFactory.createSigner(x, y, verifiersData)
+        const signerProxy = await ethers.getContractAt('SafeWebAuthnSignerProxy', await proxyFactory.getSigner(x, y, verifiersData))
```

Comment:
I'm not sure I understand the rationale behind configuring a `precompile` address for benchmarking. I would revert this change.